resolwe_bio/processes/variant_calling/create_genomicsdb.py

```diff
@@ -0,0 +1,198 @@
+
+        database = DirField(label="GenomicsDB workspace")
+        intervals = FileField(label="Intervals file")
+        species = StringField(label="Species")
+        build = StringField(label="Build")
+
+    def run(self, inputs, outputs):
+        """Run analysis."""
+        database_folder = "database"
+        sample_map_file = "sample_map.txt"
+
+        species = inputs.gvcfs[0].output.species
+        if any(gvcf.output.species != species for gvcf in inputs.gvcfs):
+            self.error("Not all of the input samples are of the same species.")
+
+        build = inputs.gvcfs[0].output.build
+        if any(gvcf.output.build != build for gvcf in inputs.gvcfs):
+            self.error("Not all of the input samples have the same genome build.")
+
+        with open(sample_map_file, "w") as sample_map:
+            for gvcf in inputs.gvcfs:
+                sample_map.write(f"{gvcf.entity_name}\t{gvcf.output.vcf.path}\n")
+
+        if inputs.use_existing and inputs.existing_db is None:
+            self.error(
+                "GATK GenomicsDB object has to be provided to add GVCFs to the existing "
+                "database."
+            )
+
+        elif inputs.use_existing and inputs.existing_db:
+            if species != inputs.existing_db.output.species:
+                self.error("The existing database and GVCFs species differ.")
+            if build != inputs.existing_db.output.build:
+                self.error("The existing database and GVCFs build differ.")
+
+            shutil.copytree(inputs.existing_db.output.database.path, database_folder)
+
+            db_import_args = [
+                "--genomicsdb-update-workspace-path",
+                database_folder,
+            ]
+            intervals = Path(inputs.existing_db.output.intervals.path)
+
+        elif inputs.intervals:
+            db_import_args = [
+                "--genomicsdb-workspace-path",
+                database_folder,
+                "-L",
+                inputs.intervals.output.bed.path,
+            ]
+            intervals = Path(inputs.intervals.output.bed.path)
+
+        else:
+            self.error("Intervals file is required for creating a new database.")
+
+        db_import_args.extend(
+            [
+                "--sample-name-map",
+                sample_map_file,
+                "--batch-size",
+                inputs.advanced_options.batch_size,
+                "--reader-threads",
+                min(self.requirements.resources.cores, 5),
+            ]
+        )
+
+        if inputs.advanced_options.consolidate:
+            db_import_args.append("--seqBias")
+
+        return_code, _, _ = Cmd["gatk"]["GenomicsDBImport"][db_import_args] & TEE(
+            retcode=None
+        )
+        if return_code:
+            self.error("GATK GenomicsDBImport tool failed.")
+
+        output_bed = f"./{intervals.name}"
+        Path(output_bed).symlink_to(str(intervals))
+
+        outputs.intervals = output_bed
```

Comment:
It's good to have this here. However, if the existing database was prepared with the interval file, it is not required to add it again:

```
Add new samples to an existing genomicsdb workspace. 
In the incremental import case, no intervals are specified in the command because the 
tool will use the same intervals used in the initial import. Sample map is also supported for incremental import.
    gatk --java-options "-Xmx4g -Xms4g" GenomicsDBImport \
      -V data/gvcfs/mother.g.vcf.gz \
      -V data/gvcfs/father.g.vcf.gz \
      -V data/gvcfs/son.g.vcf.gz \
      --genomicsdb-update-workspace-path my_database \
      --tmp-dir /path/to/large/tmp \
  ```